targets/TARGET_WICED/WICED.md

```diff
@@ -0,0 +1,229 @@
+# How to add the WICED SDK
+
+This document summarizes how to download and integrate the WICED SDK into Mbed OS.
+
+**NOTE:** This document is intended for when the WICED SDK is extracted from Mbed OS.
+
+There are several files not provided by the WICED SDK. This is a non-exhaustive
+list of files that are currently provided by Mbed OS:
+- w_generated_mac_address.txt
+- w_generated_security_dct.h
+- w_resources.h
+- mbed_lib.json
+
+## Download the WICED SDK
+
+You can find the full instructions to download the WICED SDK with Advantech's patches
+on Advantech's Wiki:
+http://ess-wiki.advantech.com.tw/view/WISE-1530_SDK
+
+To get the WICED SDK, download WICED-Studio-5.2.0 for your platform:
+https://community.cypress.com/community/wiced-wifi/wiced-wifi-documentation
+
+Once downloaded, move the WICED SDK folder into `targets/TARGET_WICED`.
+
+For the WISE1530, download the patch from Advantech:
+`WM-BN-BM-22_SDK_5.1.x_platform_patch.zip`
+
+Once downloaded, copy the contents into `targets/TARGET_WICED/WICED`.
```

Comment:
Once downloaded, copy the contents into `targets/TARGET_WICED`.

Reply:
It really should be `targets/TARGET_WICED/WICED` (after rename). Otherwise the libraries/platforms folders don't line up with their destinations. Note there are multiple nested WICED folders.